src/Components/NavBar.test.js

```diff
@@ -11,9 +11,7 @@ describe("NavBar", () => {
   ])("child at %i is a button with %s", (index, text) => {
     const wrapper = shallow(<NavBar />);
 
-    const child = wrapper.childAt(index);
-
-    expect(child).toHaveDisplayName("button");
-    expect(child).toHaveText(text);
```

Comment:
Similar deal here, as the test is basically expecting no other children of NavBar except the three buttons.